tldr/cross_file_calls/parsers/luau.py

```
@@ -0,0 +1,75 @@
+                        'column': line.find(match.group(0))
+                    })
+            
+            return imports
+            
+        except Exception:
+            return []
+    
+    def _extract_calls_regex(self, file_path: str) -> List[Dict]:
+        """Regex-based call extraction for Luau."""
+        try:
+            with open(file_path, 'r', encoding='utf-8') as f:
```

Ruff (0.14.11)

Unnecessary mode argument

Remove mode argument

(UP015)